oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/manual/endpoint/EndpointTraffic.java

```diff
@@ -39,7 +40,7 @@
     builder = EndpointTraffic.Builder.class, processor = MetricsStreamProcessor.class)
 @MetricsExtension(supportDownSampling = false, supportUpdate = false)
 @EqualsAndHashCode
-public class EndpointTraffic extends Metrics {
+public class EndpointTraffic extends Metrics implements Traffic {
```

Comment:
If you says this implementing `Traffic#name`, the name is exactly the `id`(as nodeType related to name logically). What is the point of doing this?

Reply:
After discussion, let's use `@Metadata` to separate this from real metrics.